### DISCUSSION TITLE
iOS 14: Location permissions / API updates in Capacitor/Location Plugin

### DISCUSSION THREAD

**Discussion by chrisweight** [ionic-team/capacitor#3231] [Ideas]
Hi all!

Based upon the latest iOS 14 location permissions structures and API updates (https://developer.apple.com/videos/play/wwdc2020/10660/), I'm just wondering if updating this plugin is on your roadmap at all - and if so, what's the timeframe? We develop a large number of apps that use location and the incoming changes are fairly profound!

Any information would be much appreciated!

Cheers!

**Comment by jcesarmobile** [comment]


**Reply by chrisweight** [reply]
Hey! So, since, you guys opted in to use Discussions - I figured I'd...you know..._discuss_ it with you first, to:

- save a load of work up-front that you might already have done, or have scheduled in to do.
- save opening a ticket that you then need to triage, back-and-forward on and so on, thus creating more work.

I'll take your response as a 'no, looking at this isn't on our roadmap'.

**Reply by joewoodhouse** [reply]
I think Chris is just trying to highlight to you (in a helpful way) that there are upcoming changes in iOS14 that are going to require Capacitor changes. I guess it's a matter of debate whether this counts as an "idea" or not - if the way location is fundamentally changing in iOS14 then the capacitor plugin will, I assume, need to reflect that. 

I'm sure we can all agree that we don't want to be discovering things like this a few days before iOS14 drops to the world so I would have thought some heads-up like this would be welcomed.
